server/src/main/scala/org/dbpedia/extraction/server/resources/TemplateStatistics.scala

```diff
@@ -170,7 +171,7 @@ class   TemplateStatistics(@PathParam("lang") langCode: String, @QueryParam("p")
             var redirectMsg = ""
             for (redirect <- targetRedirect) {
               if (mappingStat.isMapped) {
-                  //redirectMsg = " NOTE: the mapping for " + wikiDecode(redirect, language).substring(createMappingStats.templateNamespace.length()) + " is redundant!"
+                  redirectMsg = " NOTE: the mapping for " + redirect.substring(manager.templateNamespace.length) + " is redundant!"
```

Comment:
I also tested it locally and tried a link but links to redirected templates statistics just throw an exception. So it should be a link to the mapping page which is not really consistent with the column content... 
It was a quick fix to just make the current code work, I'll have a look at integrating it more smoothly in the page.